src/loaders/omeZarrReader.js

```diff
@@ -0,0 +1,51 @@
+import { openArray } from 'zarr';
+import ZarrLoader from './zarrLoader';
+
+async function getJson(path, key) {
+  const res = await fetch(`${path}${key}`);
+  if (res.status === 404) {
```

Comment:
Maybe also add a check for `.ok` as well.

Reply:
if the status isn't ok it should fail loudly when trying to call `res.json()`. I do this check to see if the resource exits.